wscript

```diff
@@ -312,7 +312,7 @@ iconv support use --disable-iconv.",
     }, {
         'name': 'libass',
         'desc': 'SSA/ASS support',
-        'func': check_pkg_config('libass', '>= 0.12.1'),
+        'func': check_pkg_config('libass', '>= 0.12.2'),
         'req': True,
```

Comment:
When was 0.12.1 and 0.12.2?

Reply:
0.12.1: 07. January 2015
0.12.2: 07. May 2015

Reply:
Hm, so doesn't matter.